Devscord.DiscordFramework/Services/UserService.cs

```diff
@@ -45,5 +45,15 @@ private SocketRole GetRole(ulong roleId, DiscordServerContext server)
         {
             return Server.GetRoles(server.Id).First(x => x.Id == roleId);
         }
+
+        public Task WelcomeUser(ChannelContext channelContext, UserContext userContext, DiscordServerContext serverContext)
+        {
+            var messageService = new MessagesService()
+            {
+                DefaultChannelId = channelContext.Id
```

Comment:
możesz pomyśleć czy dałoby się to jakoś zautomatyzować 

może MessagesService i inne serwisy tego typu powinny być tworzone już wcześniej, na etapie middleware 
gdzie byłyby uzupełnione o dane domyślne